hbase-assembly/pom.xml

```diff
@@ -309,6 +309,10 @@
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-zookeeper</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-hbtop</artifactId>
+    </dependency>
```

Comment:
You made a separate module for it? If so, thats great.

Reply:
Yes, I made a separate module.